doc/release-notes-bucash.md

```diff
@@ -12,11 +12,19 @@ Please report bugs using the issue tracker at github:
 This is a major release version based of Bitcoin Unlimited compatible
 with the Bitcoin Cash specifications you could find here:
 
-
 - https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/uahf-technical-spec.md (Aug 1st '17 Protocol Upgrade, bucash 1.1.0.0)
 - https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/nov-13-hardfork-spec.md (Nov 13th '17 Protocol Upgrade, bucash 1.1.2.0)
 - https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/may-2018-hardfork.md (May 15th '18 Protocol Upgrade, bucash 1.3.0.0, 1.3.0.1, 1.4.0.0)
-- https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/2018-nov-upgrade.md (Nov 15th '18 Protocol Upgrade, bucash 1.5.0.0)
+- https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/2018-nov-upgrade.md (Nov 15th '18 Protocol Upgrade, bucash 1.5.0.0, 1.5.0.1)
+
+This release will also implement a set of consensus changes proposed by an alternative implementation, Bitcoin SV,
+see [SV release notes for ver 0.1.0](https://github.com/bitcoin-sv/bitcoin-sv/blob/master/doc/release-notes.md) for more details.
+Such set of features is **disabled by default**, the default policy is to activate the set of changes as defined by the bitcoincash.org
+[specification](https://github.com/bitcoin-sv/bitcoin-sv/blob/master/doc/release-notes.md).
```

Comment:
The URLs on line 21 & 23 are the same. Is this correct? Seems like the latter should direct to a bitcoincash.org page.

Reply:
you are right indeed. fixing.